build/mac/BUILD.gn

```diff
@@ -392,7 +392,12 @@ action("sign_pkg") {
 
 action("create_dmg") {
   script = "//build/gn_run_binary.py"
-  shell_script = "//chrome/installer/mac/pkg-dmg"
+
+  if (host_os == "mac") {
+    shell_script = "//chrome/installer/mac/pkg-dmg"
+  } else {
+    shell_script = "cross-compile/pkg-dmg-linux.py"
```

Comment:
this should just be `script`

Reply:
Resolved in ac7e789b8e8a80a8623288cc34b4a6135f554345.